microcluster/app.go

```diff
@@ -480,3 +480,27 @@ func (m *MicroCluster) SQL(ctx context.Context, query string) (string, *internal
 
 	return "", batch, err
 }
+
+// APIExtensions returns the list of API extensions registered with the daemon.
+func (m *MicroCluster) APIExtensions(ctx context.Context) (map[string][]string, error) {
+	c, err := m.LocalClient()
+	if err != nil {
+		return nil, err
+	}
+
+	clusterMembers, err := c.GetClusterMembers(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(clusterMembers) == 0 {
```

Comment:
This check isn't anymore required. But instead use the `len()` of cluster members when initializing the map below.

Reply:
fixed